spec/lib/dotiw_spec.rb

```diff
@@ -327,6 +327,13 @@
             expect(distance_of_time_in_words(start, finish, true, options)).to eq(output)
           end
         end
+
+        it 'should be callable via ActionController::Base.helpers' do
+          end_time = START_TIME + 1.year + 2.months + 3.weeks + 4.days + 5.hours + 6.minutes + 7.seconds
+          expected = '1 year, 2 months, 3 weeks, 4 days, 5 hours, 6 minutes, and 7 seconds'
+          actual = ActionController::Base.helpers.distance_of_time_in_words(START_TIME, end_time, true, { vague: false })
+          expect(actual).to eq(expected)
+        end
```

Comment:
This spec duplicated my issue with the "No superclass method `distance_of_time_in_words`" error. Changing `lib/dotiw/action_view/helpers/date_helper.rb` made this spec pass.